### DISCUSSION TITLE
Add bluetooth headset battery's percentage in Ubuntu - Goulin Khoge

### DISCUSSION THREAD

**Discussion by giscus** [goulinkh/goulin.fr#17] [Blog]
# Add bluetooth headset battery's percentage in Ubuntu - Goulin Khoge

Showing the remaining battery of my Bluetooth headset in the status bar on Ubuntu 22.04

https://www.goulin.fr/blog/bluetooth-headset-battery-percentage-ubuntu

**Comment by MarcoProcida** [comment]
It works from me using Ubuntu 22.04 and jbl tune 205bt headset

**Comment by salaeng** [comment]
Thanks for that, works for me also on 22.04. First time I edited a /etc file as root admin, so I learnt that also!
